src/test/java/org/mule/extension/sftp/SftpWriteTestCase.java

```diff
@@ -113,6 +116,16 @@ public void createNewOnNotExistingParentWithoutCreateFolder() throws Exception {
     doWriteOnNotExistingParentWithoutCreateFolder(CREATE_NEW);
   }
 
+  //TODO: MULE-16515 ignore this test until issue is fixed.
+  @Test
+  @Ignore
+  public void writeOnLockedFile() throws Exception {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Fix or remove this skipped unit test ([squid:S1607](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1607))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-sftp-connector&pullRequest=61&issues=AWkMLzVPotIXnmsUtOhn&open=AWkMLzVPotIXnmsUtOhn)

[ISSUE_KEY]: AWkMLzVPotIXnmsUtOhn